### PR TITLE
TST: avoid/reduce unimportant warnings in tests

### PR DESCRIPTION
**Pull request by theroggy**

### CODE REVIEW

File: pyogrio/tests/test_geopandas_io.py [hunk 1]
```diff
@@ -243,8 +251,11 @@ def test_read_write_datetime_tz_with_nulls(tmp_path):
     assert_geodataframe_equal(df, result)
 
 
+@pytest.mark.filterwarnings("ignore: Measured")
```
Comment (jorisvandenbossche) [inline pyogrio/tests/test_geopandas_io.py:254]: Where do you see this warning? Or did it pop up by changing the layer?

File: pyogrio/tests/test_core.py [hunk 2]
```diff
@@ -451,8 +451,9 @@ def test_read_info_force_total_bounds(
         assert info["total_bounds"] is None
 
 
+@pytest.mark.filterwarnings("ignore: Measured")
 def test_read_info_without_geometry(test_fgdb_vsi):
-    assert read_info(test_fgdb_vsi)["total_bounds"] is None
+    assert read_info(test_fgdb_vsi, layer="basetable_2")["total_bounds"] is None
```
Comment (jorisvandenbossche) [inline pyogrio/tests/test_core.py:456]: What warning does this give right now?

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks for looking into this!

File: pyogrio/tests/test_core.py (hunk 2 above)
Comment (theroggy) [reply pyogrio/tests/test_core.py]: It looks like some changes related to #362 got mixed up in this PR, not sure how that happened. I rolled back the changes that belong in #362.

**Comment by theroggy** [comment]
It looks like some changes got mixed up with PR #361. Cleaned up.

File: pyogrio/tests/test_core.py [hunk 3]
```diff
@@ -451,6 +451,7 @@ def test_read_info_force_total_bounds(
         assert info["total_bounds"] is None
 
 
+@pytest.mark.filterwarnings("ignore: Measured")
```
Comment (brendan-ward) [inline pyogrio/tests/test_core.py:454]: I'm not observing this warning locally when running tests; are you still seeing it?

Comment (theroggy) [reply pyogrio/tests/test_core.py]: No indeed... removed.

**Review by brendan-ward — APPROVED** [review]

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**